runtime/src/main/java/org/corfudb/runtime/collections/CorfuRecord.java

```diff
@@ -0,0 +1,14 @@
+package org.corfudb.runtime.collections;
+
+import com.google.protobuf.Message;
+
+import lombok.Data;
+
+/**
+ * Created by zlokhandwala on 2019-08-15.
+ */
+@Data
+public class CorfuRecord<V extends Message, M extends Message> {
+    private final V payload;
```

Comment:
why payload is `V` not `P` at least?

Reply:
V was to distinguish from M. V and M stand for Value and Metadata.
This also keeps it uniform with the naming conventions used in the rest of the CorfuStore.